### PR TITLE
Explorationedits

### PR DESCRIPTION
**Pull request by Tasald**
Adds a new set of exploration armor plates. adds various items to the exploration vendor, and lowers the price of the carbine.

Will probably need discussion on items and the armor.

### CODE REVIEW

**Comment by Tasald** [comment]
gona need to get input on this probably, for the new armor, and the denecrotizer atleast.

**Comment by Tasald** [comment]
not sure why the ecig thing is still in this.. it shouldnt be

File: code/modules/clothing/under/accessories/armor.dm [hunk 1]
```diff
@@ -97,6 +97,13 @@
 	icon_state = "armor_light"
 	armor = list(melee = 30, bullet = 20, laser = 20, energy = 20, bomb = 35, bio = 75, rad = 35)
 
+	/obj/item/clothing/accessory/armor/armorplate/explorer/adv //CHOMPADD
+	name = "explorer armor plate" //CHOMPADD
+	desc = "Advanced version of the standared exploration plate. Attaches to a plate carrier." //CHOMPADD
+	icon_state = "armor_light" //CHOMPADD
+	armor = list(melee = 45, bullet = 30, laser = 30, energy = 30, bomb = 35, bio = 75, rad = 35) //CHOMPADD
+armorsoak = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0) //CHOMPADD
```
Comment (Rykka-Stormheart) [inline code/modules/clothing/under/accessories/armor.dm:105]: Needs indented to match.

File: code/modules/mining/ore_redemption_machine/survey_vendor.dm [hunk 2]
```diff
@@ -54,7 +54,9 @@
 		EQUIPMENT("Defense Equipment - Plasteel Machete",		/obj/item/weapon/material/knife/machete,							50),
 		EQUIPMENT("Defense Equipment - Razor Drone Deployer",	/obj/item/weapon/grenade/spawnergrenade/manhacks/station/locked,	100),
 		EQUIPMENT("Defense Equipment - Sentry Drone Deployer",	/obj/item/weapon/grenade/spawnergrenade/ward,						150),
-		EQUIPMENT("Defense Equipment - Frontier Carbine",		/obj/item/weapon/gun/energy/locked/frontier/carbine,				750),
+		EQUIPMENT("Defense Equipment - Frontier Carbine",		/obj/item/weapon/gun/energy/locked/frontier/carbine,				200), //CHOMPEDIT
+		EQUIPMENT("Defense Equipment - Frontier Phaser",		/obj/item/weapon/gun/energy/locked/frontier,						50), //CHOMPADD
+		EQUIPMENT("Defense Equipment - Holdout Phaser", 		obj/item/weapon/gun/energy/locked/frontier/holdout,				75), //CHOMPADD
```
Comment (Rykka-Stormheart) [inline code/modules/mining/ore_redemption_machine/survey_vendor.dm:59]:
```suggestion
		EQUIPMENT("Defense Equipment - Holdout Phaser", 		/obj/item/weapon/gun/energy/locked/frontier/holdout,				75), //CHOMPADD
```

File: code/modules/clothing/under/accessories/armor.dm [hunk 3]
```diff
@@ -279,6 +292,12 @@
 	icon_state = "legguards_green"
 	armor = list(melee = 30, bullet = 20, laser = 20, energy = 20, bomb = 35, bio = 75, rad = 35)
 
+	/obj/item/clothing/accessory/armor/legguards/explorer/adv //CHOMPADD
```
Comment (Rykka-Stormheart) [inline code/modules/clothing/under/accessories/armor.dm:295]:
```suggestion
/obj/item/clothing/accessory/armor/legguards/explorer/adv //CHOMPADD
```

**Review by Rykka-Stormheart — CHANGES_REQUESTED** [review]
These things need correcting for your PR to pass checks, but otherwise it appears fine.

**Comment by Cyrelius** [comment]
You made the weapons too cheap in the vendor, and they should be at least double.

Regarding the armour, we need to have an internal discussion, as those plates you add are practically as good as armor that security gets in the lethal section of the **armoury**, but at the same time, I consider what's in the armoury to be trash.  It definitely can't go in without something else changing.  That being said, I wouldn't mind them being able to get better armour from the vendor, but we'll have to see on how good that'll be.

The denecrotizer is a bad idea.  That's something that medical should possibly have mapped in, and science should be able to make, and shouldn't be in a vendor at all, much less one restricted to only explorers.

File: code/modules/clothing/under/accessories/armor.dm [hunk 4]
```diff
@@ -97,6 +97,13 @@
 	icon_state = "armor_light"
 	armor = list(melee = 30, bullet = 20, laser = 20, energy = 20, bomb = 35, bio = 75, rad = 35)
 
+	/obj/item/clothing/accessory/armor/armorplate/explorer/adv //CHOMPADD
```
Comment (BlackMajor) [inline code/modules/clothing/under/accessories/armor.dm:100]: You've got a tab here that you shouldn't

**Comment by BlackMajor** [comment]
Also since we have a modular folder now, I'm requesting you move the new armor you've put into 'code/modules/clothing/under/accessories/armor.dm' and create a file under 'modular_chomp/code/modules/clothing/under/accessories/armor.dm' for them. No need for chomp comments since it's the modular folder.

**Comment by Tasald** [comment]
made changes from staff comments, not putting the new folder yet, going to wait until I get what stats are ok for it.

**Comment by Razgriz1032** [comment]
I don't think we'll include the armor in this tho. As soon as the leg guards is fixed, we'll merge it now. The new types of armor, I don't think we'll include in this and it'd need like a separate project.

**Pull request merged**